dash_cytoscape/Cytoscape.py

```diff
@@ -12,8 +12,8 @@ class Cytoscape(Component):
 - id (string; optional): The ID used to identify this component in Dash callbacks.
 - className (string; optional): Sets the class name of the element (the value of an element's html
 class attribute).
-- style (dict; optional): Add inline styles to the root element.
-- elements (list; optional): A list of dictionaries representing the elements of the networks.
+- style (dict; default {width: '600px', height: '600px'}): Add inline styles to the root element.
```

Comment:
Was this manually added, or did you regenerate `Cytoscape.py`?

Reply:
Auto generated